index.html

```diff
@@ -32,6 +33,15 @@ <h1>Task List ↴ </h1>
           </form>
           </header>
     </div>
+
+
+    <div class="alertt hide" id="custAlert3">
+      <span class="msgt">Task added Successfully</span>
```

Comment:
Add following code in between
```html
<span class="fas fa fa-check-circle" style="font-size:24px"></span>
```      

| Prev: | After update: |
|---|---|
| ![image](https://github.com/malivinayak/To-Do-Manager/assets/66154908/5e80ec84-001b-4966-adf8-b2a663864f31) | ![image](https://github.com/malivinayak/To-Do-Manager/assets/66154908/86bc969b-439d-4ad9-81d3-0b5f49418cac) |

Reply:
In after update screenshot tock mark is added which looks more appropriate.

Reply:
![image](https://github.com/malivinayak/To-Do-Manager/assets/116938919/fdf32770-7138-4269-88bf-17c6e55c9edf)